internal/awstesting/client.go

```diff
@@ -7,7 +7,7 @@ import (
 // NewClient creates and initializes a generic service client for testing.
 func NewClient(cfg aws.Config) *aws.Client {
 	if cfg.Retryer == nil {
-		cfg.Retryer = aws.DefaultRetryer{NumMaxRetries: 3}
+		cfg.Retryer = aws.NewDefaultRetryer()
```

Comment:
This should prevent the need for adding the `cfg.Retryer = aws.NewDefaultRetryer()` in the above tests when no special configuration is being specified.

Reply:
Removing retryer initialization from awstesting as aws/client.go has NewClient method to initialize retryer with default retryer if not specified